### PR TITLE
4 icons

### PR DESCRIPTION
**Pull request by wuku0**
## Description

Added 4 new icons

## Type of change

:x: Bug fix (non-breaking change which fixes an issue)
:white_check_mark: Icon addition (non-breaking change that adds/modifies Lawnicons's icons)
:x: General change (non-breaking change that doesn't fit the above categories, such as copyediting)

### Requests
* Fuehrerschein (`de.adac.Fuehrerschein/de.adac.Fuehrerschein.MainActivity`)
* Playtomic (`com.playtomic/com.playtomic.general.LaunchScreenActivity`) - added from icon request form
* Mauth (`com.xinto.mauth.debug/com.xinto.mauth.ui.MainActivity`)
* easyJet (`com.mttnow.droid.easyjet/com.mttnow.droid.easyjet.ui.home.Splash.Activity`)

### CODE REVIEW

File: svgs/mauth.svg
Comment (Chefski) [inline svgs/mauth.svg:1]: Please change the icon to use an outline style

File: svgs/fuehrerschein.svg
Comment (Chefski) [inline svgs/fuehrerschein.svg:1]: Please change the icon to use an outline style

File: svgs/easyjet.svg
Comment (Chefski) [inline svgs/easyjet.svg:1]: Please change the icon to use an outline style

File: svgs/playtomic.svg
Comment (Chefski) [inline svgs/playtomic.svg:1]: Please change the icon to use stroke paths and rounded corners

**Review by Chefski — CHANGES_REQUESTED** [review]
Please resize all the icons to the 160x160 content area and check comments

File: svgs/fuehrerschein.svg
Comment (KTibow) [reply svgs/fuehrerschein.svg:1]: i'm confused where this icon came from - [official one for reference](https://play.google.com/store/apps/details?id=de.adac.Fuehrerschein)

File: svgs/easyjet.svg
Comment (KTibow) [reply svgs/easyjet.svg:1]: i'm confused where this icon came from - [official one for reference](https://play.google.com/store/apps/details?id=com.mttnow.droid.easyjet)

**Pull request closed without merging**